backend/WebUI/api_webui.go

```diff
@@ -530,7 +551,7 @@ func GetTenants(c *gin.Context) {
 	setCorsHeader(c)
 
 	if !CheckAuth(c) {
-		c.JSON(http.StatusNotFound, bson.M{})
+		c.JSON(http.StatusNotFound, gin.H{"cause": "Illegal Token"})
```

Comment:
I think `http.StatusUnauthorized` would be more appropriate here.

Reply:
Or `http.StatusForbidden`, but that depends on why exactly `CheckAuth` failed. If it failed because the token is expired or invalid, it should be `http.StatusUnauthorized`. If it failed because the token does not include the required claims, it should be `http.StatusForbidden`.